consistency/tool/comparison/comparison.py

```diff
@@ -46,14 +41,23 @@ def find_closest_id(word):
         return closest_word_groups
 
     def find_word_group(self):
-        # TODO: using the map returned by find_word_group,
-        # if the vocab has an associated ID, form a Variation object
-        # add this object into the current_variations list
-        # if not, check if the vocab is a NOISE_WORD, if not, add it the
-        # unique_terms list object. If there is a matching word, form a variation object
-        # from the match -> means creating a wordgroup instance at the dot.
-        # return the current variations object
-        return
+
```

Comment:
This function looks incomplete, maybe this part should be added in a separate PR. And can we also rename this to something like `generate_consistency_report`? 

`find_word_group` and `find_closest_word_groups` have similar meanings.